mivs/models.py

```diff
@@ -0,0 +1,205 @@
+from mivs import *
+
+
+@Session.model_mixin
+class SessionMixin:
+    def logged_in_studio(self):
+        try:
+            return self.indie_studio(cherrypy.session['studio_id'])
+        except:
+            raise HTTPRedirect('../mivs_applications/start')
+
+    def delete_screenshot(self, screenshot):
+        self.delete(screenshot)
+        try:
+            os.remove(screenshot.filepath)
+        except:
+            pass
+        self.commit()
+
+
```

Comment:
Uber plugins can actually modify our built-in database tables.  In this case we don't actually need to add any new columns or anything like that, but we do want to define a new relationship, since indie judges will be a type of admin account and should therefore be linked to this table.